packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -111,7 +115,7 @@ export class TruncatedFormat extends React.Component<ITruncatedFormatProps, ITru
                     <div className={Classes.TABLE_TRUNCATED_VALUE} ref={this.handleContentDivRef}>{cellContent}</div>
                     <Popover
                         className={Classes.TABLE_TRUNCATED_POPOVER_TARGET}
-                        constraints={constraints}
+                        tetherOptions={tetherOptions}
```

Comment:
Good idea, i've implemented this for truncatedFormat.tsx and columnHeaderCell.tsx. Do I need to do this for contextMenu.tsx too or did you leave this out intentionally?